.github/workflows/main.yml

```diff
@@ -98,6 +100,164 @@ jobs:
 
       - name: Test Netbeans Build System
         run: ant -Dcluster.config=basic localtest
+
+  linux-php:
+    name: Linux build of PHP modules
+    runs-on: ubuntu-18.04
+    env:
+      OPTS: -quiet -Dcluster.config=php -Djavac.compilerargs=-nowarn -Dbuild.compiler.deprecation=false -Dtest-unit-sys-prop.ignore.random.failures=true
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Caching dependencies
+        uses: actions/cache@v2
+        with:
+          path: ~/.hgexternalcache
+          key: ${{ runner.os }}-${{ hashFiles('**/external/binaries-list') }}
+          restore-keys: ${{ runner.os }}-
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@2.3.1
+        with:
+          php-version: '7.4'
+          tools: pecl
+          extensions: xdebug
+          ini-values: xdebug.remote_enable=on
+
+      - name: Clean
+        run: ant $OPTS clean
+        
+      - name: Build
+        run: ant $OPTS build
+        
+      - name: Test Hudson
+        uses: GabrielBB/xvfb-action@v1
+        with:
+          run: ant ${{ env.OPTS }} -f php/hudson.php test
+
+      - name: Test Neon
+        uses: GabrielBB/xvfb-action@v1
+        with:
+          run: ant ${{ env.OPTS }} -f php/languages.neon test
```

Comment:
@KacerCZ Could you try to run Xvfb in the first steps inteads use the `GabrielBB/xvfb-action@v1` action?
For example: https://github.com/apache/netbeans/pull/2186/files
```yaml
  linux:
    name: Linux build
    runs-on: ubuntu-18.04
    env:
      DISPLAY: ":99.0"
    steps:
      - name: Setup JDK 8
        uses: actions/setup-java@v1
        with:
          java-version: '8'
          java-package: jdk+fx

      - name: Launch Xvfb
        run: Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
```

Reply:
Changed.